kubernetes-manifests/cartservice.yaml

```diff
@@ -51,12 +51,14 @@ spec:
           value: "http://$(NODE_IP):9943/v2/datapoint"
         - name: SIGNALFX_PROFILER_ENABLED
           value: "true"
+        - name: SIGNALFX_PROFILER_MEMORY_ENABLED
+          value: "true"
+        - name: SIGNALFX_METRICS_Process_ENABLED
```

Comment:
Why is it not capitalized?

Reply:
This is the way how we have constructed possibility to disable/enalbe instrumentation both for metrics and traces (case sensitive).

Reply:
It follows the pattern established by Java instrumentations that use this pattern with the "cased" name of the instrumentation (traces and metrics) to be enabled/disabled.